torch_geometric/transforms/SKLearnPreprocessTransform.py

```diff
@@ -0,0 +1,40 @@
+from typing import List, Union
+
+import torch
+
+from torch_geometric.data import Data, HeteroData
+from torch_geometric.data.datapipes import functional_transform
+from torch_geometric.transforms import BaseTransform
+
+
+@functional_transform('sklearn_preprocess')
+class SKLearnPreprocessTransform(BaseTransform):
```

Comment:
Yup, will do. I'll add the test. I'm thinking of modifying the approach in this transform, so that it doesn't need the hack to send in an edge scaler and is agnostic to any sklearn method or pipeline. I got busy with another bug i was dealing with, but now i can work on this